packages/webview_flutter/webview_flutter_wkwebview/pubspec.yaml

```diff
@@ -18,6 +18,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
+  path: ^1.8.0
```

Comment:
We'll need to keep an eye on this over time; we may need to re-implement our own version of the functionality we need. This is one of the packages Flutter pins, so we'll risk strong coupling of plugin versions to Flutter versions (i.e., if Flutter version X changed to `path: 2.0` we'd have to either lock the versions, or do an update to allow either 1.x or 2.0, if possible).

It's not too likely, so I think we can do this for now, but something we'll need to be aware of as something we might have to revisit suddenly.

Reply:
I agree. Does it change anything if path is only a `dev_dependency`, though: https://github.com/flutter/flutter/blob/master/packages/flutter/pubspec.yaml#L31

`flutter_test` and `integration_test` have it as a `dependency`, so I would guess it wouldn't.

Reply:
That's a good point, if it's all only transitively through `dev_dependency` it would presumably only affect our CI, not package clients.